xcm/src/lib.rs

```diff
@@ -436,10 +436,10 @@ impl WrapVersion for AlwaysV2 {
 }
 
 /// `WrapVersion` implementation which attempts to always convert the XCM to the latest version before wrapping it.
-pub type AlwaysLatest = AlwaysV1;
+pub type AlwaysLatest = AlwaysV2;
 
 /// `WrapVersion` implementation which attempts to always convert the XCM to the release version before wrapping it.
-pub type AlwaysRelease = AlwaysV0;
+pub type AlwaysRelease = AlwaysV2;
```

Comment:
it seems to me like we already moved to v2 somewhere, but forgot to update these? If so, can we have a test that would prevent this again?

Reply:
I think you are talking about the safe XCM version or the `CurrentXcmVersion` in the XCM pallet, which is different from the `AlwaysLatest` and `AlwaysRelease` type aliases here. This type here is primarily used in the configs of the `ChildParachainRouter` and the XCMP queue pallet for the purpose of version negotiation. I believe nowadays chains use the XCM pallet as a default, but people can still freely choose to substitute the XCM pallet with these two types as a quick way to simply mandate all incoming and outgoing XCM versions to be of a particular version, instead of discovering and negotiating with the destination chain.